src/main/scala/org/constellation/concurrency/cuckoo/MutableCuckooFilter.scala

```diff
@@ -0,0 +1,67 @@
+package org.constellation.concurrency.cuckoo
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+
+import cats.effect.Sync
+import net.cinnom.nanocuckoo.NanoCuckooFilter
+import org.constellation.schema.snapshot.FilterData
+import cats.syntax.all._
+
+class MutableCuckooFilter[F[_], T](filter: NanoCuckooFilter)(implicit sync: Sync[F]) {
+
+  def insert(item: T)(implicit convert: T => String): F[Boolean] = sync.delay {
```

Comment:
ideally, every operation should be delayed separately
https://typelevel.org/cats-effect/docs/2.x/datatypes/io#keep-granularity

Reply:
Ok, I will fix it.